.travis.yml

```diff
@@ -14,27 +14,36 @@ branches:
 
 sudo: true
 
-php:
-  - 7.0
-  - 7.1
-  - hhvm
-
 matrix:
   fast_finish: true
   include:
+    - php: 7.0
+      env: ES_VERSION="5.5" TEST_BUILD_REF="origin/5.5"
+    - php: 7.0
+      env: ES_VERSION="6.0" TEST_BUILD_REF="origin/6.0"
+    - php: 7.0
+      env: ES_VERSION="6.x" TEST_BUILD_REF="origin/6.x"
+
     - php: 7.1
-      env: ES_VERSION="5.0" TEST_BUILD_REF="origin/5.0"
+      env: ES_VERSION="5.5" TEST_BUILD_REF="origin/5.5"
     - php: 7.1
-      env: ES_VERSION="5.1" TEST_BUILD_REF="origin/5.1"
+      env: ES_VERSION="6.0" TEST_BUILD_REF="origin/6.0"
     - php: 7.1
-      env: ES_VERSION="5.2" TEST_BUILD_REF="origin/5.2"
+      env: ES_VERSION="6.x" TEST_BUILD_REF="origin/6.x"
+
+    - php: hhvm
+      env: ES_VERSION="5.5" TEST_BUILD_REF="origin/5.5"
+    - php: hhvm
+      env: ES_VERSION="6.0" TEST_BUILD_REF="origin/6.0"
+    - php: hhvm
+      env: ES_VERSION="6.x" TEST_BUILD_REF="origin/6.x"
+
   allow_failures:
     - php: hhvm
+    - env: ES_VERSION="6.x" TEST_BUILD_REF="origin/6.x"
```

Comment:
Ah, I see what you mean.  I'm of split opinion: having builds fail on 6.x because of a new feature, change in API, etc is a convenient way to know that upstream has added something.  That said, I agree it can be frustrating for outsiders because PRs will fail for unrelated reasons.

Let's go ahead and mark 6.x as `allow_failures` :)